skunkworksza/www
# lab testing (#20)

* Create static.yml

* Create index.html

* Create course-catalog.html (#3)

asdada

* Create about-us.html (#4)

* Create contact.html

* Create onboarding.html (#5)

* Update index.html

* Create styles.css (#7)

* Create parallax.css (#8)

* Create parallax.js

* Create README.md

* Update issue templates (#6)

* Create skunkworks-landing-page.html

* Update README.md

* Update README.md

* Update styles.css

* Update index.html (#11)

* Update skunkworks-landing-page.html

* Update index.html

* Create main.css (#13)

* Create script.js

* Update about-us.html

* Update contact.html

* Delete skunkworks-landing-page.html

* Create CNAME

* Create github-pages.yml (#14)

* deploy: fe8c94beee680e1755f8876fc7f5eee45ea4e7af

* Create CNAME

* Delete .github/workflows/static.yml (#15)

* deploy: 61dcdd658aefd3ba294009665b8179f4f0a478b4

* Create jekyll-gh-pages.yml

* changes

* Update

* Update

* Update

* Update

* Update

* Update course-catalog.html

* Update index.html

* Update contact-us.html

* Update

* Update about-us

* Update about-us.html

* Update course-catalog.html

* Update

* Update

* Update contact.html

* Update

* Update onboarding.html

* Update contact.html

* Update contact.html

* Update contact.html

* Update index.html

* Update onboarding.html

* Update onboarding.html

* Update onboarding.html

* Update

* Update course-catalog.html (#17)

* Update course-catalog.html

* Update course-catalog.html

* landing page update

* course-catalog page updated

* contact-us deleted

* onboarding form updated

* x

* added outerboxx.html

* Update Outerbox.xhtml (#18)

Update Outerbox.html

* Update Outerbox.html

* Update Outerbox.html

* testing

* commit changes to index.html

* Update index.html

* Create ibm-security-verify-access.html (#19)

---------
